### PR TITLE
Update bug_report.md

### PR DESCRIPTION
**Pull request by mehabhalodiya**
## Proposed changes

Resolves:
- #3357 

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Not any!

### CODE REVIEW

**Review by ispeakc0de — APPROVED** [review]

**Comment by mehabhalodiya** [comment]
Can you @ispeakc0de  please let me know, is there anything left to do from my side?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ labels: bug
 
 **What you expected to happen**:
 
+**Where is this stated?**
```
Comment (neelanjan00) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]:
```suggestion
**Where can this issue be corrected? (optional)**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -20,6 +20,9 @@ labels: bug
...
+<!-- provide additional information about the location that needs to be modified -->
```
Comment (neelanjan00) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]:
```suggestion
<!-- provide additional information about the file location or directory that needs to be modified to fix this issue, if available -->
```

**Review by neelanjan00 — CHANGES_REQUESTED** [review]

**Review by neelanjan00 — APPROVED** [review]

**Comment by mehabhalodiya** [comment]
@rajdas98 Can you PTAL and merge it, if possible?
Thank you.

**Pull request merged**